keyboards/handwired/jscotto/scotto40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2022 Joe Scotto
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+// Tap Dance declarations
```

Comment:
Could the tapdance stuff be moved to another keymap? 

We want to keep the default keymap fairly simple, as it's cloned when making a new keymap.  For newer users, complicated keymaps tend to cause more issues than they solve. 

That said, adding a "stock" type keymap with more features is fine.

Reply:
The TD on default is specifically for CMD+SPC and ESC which have nowhere to go on the board, that's why it's there.

Reply:
Setting the default keymap to use the `LAYOUT_ortho_4x10` macro instead would give you a few extra keys.

Reply:
The intended mapping is the one I have set, that is just an alternative option the board offers.